### DISCUSSION TITLE
Seeking help for non-parametric analogue to 2-way ANOVA with interaction

### DISCUSSION THREAD

**Discussion by laurahspencer** [RobertsLab/resources#1224] [Q&A]
Seeking some help on what I think _should_ be a very simple statistical question. 

I have various response variables (continuous) that I'd like to compare amongst two factors, including their interactions (temperature, food, temperature x food). Sample sizes are small (n=4 per treatment) but equal across treatments. For those response variables that meet assumptions of normality I am using 2-way ANOVA. But what do I use for those that are non-parametric, and which I am unable to transform to a normal distribution?  

Is it valid to transform continuous variables to rank-order then examine via 2-way ANOVA? 

Is it valid to use multiple Kruskal-Wallis tests (non-parametric analogue to 1-way ANOVA) in this way with multiple comparison corrections?: 
- 1) test main effect of independent factor 1 (temperature)
- 2) test main effect of independent factor 2 (food)
- 3) test for interactions by creating a 3rd "treatment" variable (reflecting both temperature & food)

I have explored the response variables to try to identify their distributions and use glms with Analysis of Deviance tables, but with such little data I'm not confident in that approach either. 

Google hasn't solved this problem for me yet, so I'm hoping one of you has some insight. As promised, pub-a-thon points will be divided 60/40. Thanks!

**Comment by sr320** [comment]
via Brent...

>I would first look at the distribution of residuals—to see how off they are.  Dependent variables do not have to be normally distributed .

**Reply by laurahspencer** [reply]
Thanks- One of the dependent variables just won't behave, but there is a pretty obvious effect of food level (see figure). 

![image](https://user-images.githubusercontent.com/17264765/119693409-dc9e1080-be11-11eb-8f5b-e5228e62c1a8.png)

I found this approach - "[Two-way Anova with Robust Estimation](https://rcompanion.org/rcompanion/d_08a.html)" - seems like a pretty decent way to approximate effects?  Here's what I came up with: 

Larval release onset was first examined with Two-way ANOVA with Robust Estimation using pbad2way() from the WRS2 package for R (est=”mom” and nboot=5000, Mair & Wilcox 2020), then the significant main effect (simulated p-value < 0.05) was further examined with a Kruskal Wallis rank sum test.
